CMakeLists.txt

```diff
@@ -40,6 +40,9 @@ option(ENABLE_COVERAGE "Enable code coverage (with GCC)" Off)
 option(ENABLE_PEDANTIC_WARNINGS "Enable pedantic compiler warnings" On)
 option(ENABLE_TOOLS "Enable Umpire development tools" Off)
 
+# Set benchmark installation to OFF
+set(BENCHMARK_ENABLE_INSTALL OFF)
```

Comment:
This option will prevent Google Benchmark targets from being installed once BLT contains a newer version of GoogleTest - this is pending.